cocotb/share/makefiles/simulators/Makefile.xcelium

```diff
@@ -103,9 +105,22 @@ else
     $(error "A valid value (verilog or vhdl) was not provided for TOPLEVEL_LANG=$(TOPLEVEL_LANG)")
 endif
 
+define make_lib
```

Comment:
xrun accepts the "-makelib mylibname mysource1.vhd ... -endlib" options which should work. No need for cds.lib.